docs/Dockerfile

```diff
@@ -1,5 +1,5 @@
 # version 9.5.2
 FROM squidfunk/mkdocs-material@sha256:9919d6ee948705ce6cd05e5bc0848af47de4f037afd7c3b91e776e7b119618a4
 
-ADD requirements.txt /tmp/
+COPY requirements.txt /tmp/
```

Comment:
To be honest, in this case there is no difference between ADD and COPY, they both work the same way, the difference is that ADD can unpack files and accept remote URLs, while COPY cannot.

To be more "docker-efficient" this should use [Bind mounts](https://docs.docker.com/build/building/best-practices/#add-or-copy) for temporary `RUN` commands, but that's completely irrelevant in this case where it's a single build for something that's ephemeral/temporary/local.

Approving this PR, as this change makes sense to be more compliant with Sonar.